src/encoded/static/components/rnaget.js

```diff
@@ -93,6 +93,22 @@ AtIdRenderer.propTypes = {
 RNAGetDataRegistry.register({ field: '@id' }, AtIdRenderer);
 RNAGetDataRegistry.register({ field: 'libraryPrepProtocol' }, AtIdRenderer);
 RNAGetDataRegistry.register({ field: 'expressionID' }, AtIdRenderer);
+RNAGetDataRegistry.register({ field: 'analysis' }, AtIdRenderer);
+
+
+/**
+ * @id cell renderer that displays links to genes from the RNAGet API.
+ */
+const RNAGetIdRenderer = ({ value }) => (
+    <a href={`/genes/${value.split('/')[1]}`}>{value.split('/')[0]}</a>
+);
+
+RNAGetIdRenderer.propTypes = {
+    /** @id value to render */
```

Comment:
You cannot use a term to define the same term.

Reply:
The splitting in the renderer a little confusing. Doesn't actually seem like a normal `@id`.